devito/ops/compiler.py

```diff
@@ -14,6 +14,9 @@ class CompilerOPS(configuration['compiler'].__class__):
     def __init__(self, *args, **kwargs):
         kwargs['cpp'] = True
         self._ops_install_path = os.environ.get('OPS_INSTALL_PATH')
+        if not self._ops_install_path:
+            raise ValueError("Couldn't find OPS_INSTALL_PATH \
+                environment variable, please check your OPS installation")
```

Comment:
We are temporarily replacing raise error with warning.

Reply:
ok ! thanks. fixing